sql/topology/nodeNetwork.sql

```diff
@@ -31,9 +31,9 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 
 --v2.6
-CREATE FUNCTION pgr_nodeNetwork(
-  TEXT, -- edge table (required)
-  DOUBLE PRECISION, -- tolerance (required)
+CREATE OR REPLACE FUNCTION pgr_nodeNetwork(
+  edge_table TEXT, -- edge table (required)
```

Comment:
pgRouting policy is that named parameters are on optional parameters, compulsory parameters are not named
```
TEXT,
DOUBLE PRECISION,
```